rllib/env/tests/test_multi_agent_env.py

```diff
@@ -332,7 +333,13 @@ def compute_actions(
                 timestep=None,
                 **kwargs
             ):
-                return [0] * len(obs_batch), [[h] * len(obs_batch)], {}
+                obs_shape = (len(obs_batch),)
+                actions = np.zeros(obs_shape, dtype=np.int32)
+                states = tree.map_structure(
```

Comment:
In case of batched output state, the batch_dim should be inside the nested structure not outside.